tools/webpack-core/src/test/RequireMagicCommentsPlugin.test.ts

```diff
@@ -97,7 +97,7 @@ describe("RequireMagicCommentsPlugin", () => {
     expect(fs.readFileSync(path.join(__dirname, "dist/main.js"), "utf8")).to.matchSnapshot();
   });
 
-  it.skip("should work with copyfile comments and addCopyFileSuffix handler", async () => {
+  it("should work with copyfile comments and addCopyFileSuffix handler", async () => {
```

Comment:
@wgoehrig @calebmshafer @aruniverse ?
```
  1) RequireMagicCommentsPlugin
       should work with copyfile comments and addCopyFileSuffix handler:

      AssertionError: expected value to match snapshot RequireMagicCommentsPlugin should work with copyfile comments and addCopyFileSuffix handler 1
      + expected - actual

       exports.id = 179;
       exports.ids = [179];
       exports.modules = {

      -/***/ 993:
      +/***/ 861:
       /***/ ((module, __unused_webpack_exports, __webpack_require__) => {

       module.exports = (__webpack_require__(17).resolve)(__dirname, __webpack_require__.p + "static/bar.d9b9bf.txt");

--

       /***/ 67:
       /***/ ((__unused_webpack_module, __unused_webpack_exports, __webpack_require__) => {

      -__webpack_require__(/*webpack: copyfile*/993);
      +__webpack_require__(/*webpack: copyfile*/861);

       /***/ }),

       /***/ 17:

      at Context.<anonymous> (lib\test\RequireMagicCommentsPlugin.test.js:96:114)

  2) RequireMagicCommentsPlugin
       should work with copyfile comments, addCopyFileSuffix, and resolve converted:

      AssertionError: expected value to match snapshot RequireMagicCommentsPlugin should work with copyfile comments, addCopyFileSuffix, and resolve converted 1
      + expected - actual

       exports.id = 179;
       exports.ids = [179];
       exports.modules = {

      -/***/ 993:
      +/***/ 861:
       /***/ ((module, __unused_webpack_exports, __webpack_require__) => {

       module.exports = (__webpack_require__(17).resolve)(__dirname, __webpack_require__.p + "static/bar.d9b9bf.txt");

--

       /***/ 67:
       /***/ ((__unused_webpack_module, __unused_webpack_exports, __webpack_require__) => {

      -__webpack_require__(/*webpack: copyfile*/993);
      +__webpack_require__(/*webpack: copyfile*/861);

       /***/ }),

       /***/ 17:

      at Context.<anonymous> (lib\test\RequireMagicCommentsPlugin.test.js:109:114)

  3) RequireMagicCommentsPlugin
       should work with copyfile comments, addCopyFileSuffix, and resolve not converted:

      AssertionError: expected value to match snapshot RequireMagicCommentsPlugin should work with copyfile comments, addCopyFileSuffix, and resolve not converted 1
      + expected - actual

       exports.id = 179;
       exports.ids = [179];
       exports.modules = {

      -/***/ 993:
      +/***/ 861:
       /***/ ((module, __unused_webpack_exports, __webpack_require__) => {

       module.exports = (__webpack_require__(17).resolve)(__dirname, __webpack_require__.p + "static/bar.d9b9bf.txt");

--

       /***/ 67:
       /***/ (() => {

      -/*require.resolve*/(/*webpack: copyfile*/993);
      +/*require.resolve*/(/*webpack: copyfile*/861);

       /***/ }),

       /***/ 17:

      at Context.<anonymous> (lib\test\RequireMagicCommentsPlugin.test.js:121:114)
```

Reply:
This is on my todo, just need to regen these snapshot files on windows

Reply:
I'll boot up my windows and do this on your branch

Reply:
I'm on Windows ATM, if you can tell me what to do.

Reply:
Just nuke this file, `tools/webpack-core/src/test/RequireMagicCommentsPlugin.test.snap`, then run `rushx test` to regen it.

Reply:
done, thanks.